source/ocsp-support/ocsp-support.rst

```diff
@@ -451,7 +478,7 @@ disabling non-stapled OCSP via ``tlsDisableOCSPEndpointCheck`` or by
 disabling certificate revocation checking altogether
 via ``tlsDisableCertificateRevocationCheck``.
 
-An application that uses a driver that utilizes hard-fail behavior
+An application that uses a driver that utilizes hard fail behavior
```

Comment:
The changelog entry says "hard-fail", but you purposely removed the hyphen here. Should this be consistent throughout?

Reply:
Fixed! Note, currently we tend to use "hard fail" when it's in quotes and and hard-fail when it's not quoted. Ditto with "soft fail" vs soft-fail.